common/src/main/java/org/jetbrains/annotations/CheckReturnValue.java

```diff
@@ -0,0 +1,30 @@
+package org.jetbrains.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.Collection;
+
+/**
+ * Specifies that the return value of the method should be used.
+ * <p>
+ * Typical use cases are:
+ * <ul>
+ *   <li>pure or side-effect-free methods
+ *   <li>methods that use the return value for reporting success or error
+ *   <li>methods that return an unfinished computation,
+ *       such as {@link Collection#stream()} or builders
+ * </ul>
+ * <p>
+ * When used on a type, the annotation applies to all methods that do not return {@code void}.
+ * <p>
+ * When used on a package, the annotation applies to all types of that package.
+ *
+ * @see Contract#pure()
+ */
+@Target({ElementType.METHOD, ElementType.CONSTRUCTOR, ElementType.TYPE, ElementType.PACKAGE})
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
Why do you use `RUNTIME` retention for this annotation? It's explicitly forbidden in [CONTRIBUTING.md](https://github.com/JetBrains/java-annotations/blob/master/CONTRIBUTING.md#retention-policy).